auth/oras/transport.go

```diff
@@ -0,0 +1,53 @@
+package oras
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+// Transport is an http.RoundTripper that keeps track of the in-flight
+// request and add hooks to report HTTP tracing events.
+type Transport struct {
```

Comment:
Will `DebugTransport` a better name?

Reply:
Changed the type name to DebugTransport.